www/front_src/src/Resources/Details/tabs/Services/index.tsx

```diff
@@ -98,11 +99,11 @@ const ServicesTab = ({ details }: TabProps): JSX.Element => {
 
   const selectService = (service) => (): void => {
     setOpenDetailsTabId(detailsTabId);
-    setSelectedResourceUuid(service.uuid);
+    setSelectedResourceUuid(`${selectedResourceUuid}-s${service.id}`);
```

Comment:
```suggestion
    setSelectedResourceUuid(`h${selectedResourceId}-s${service.id}`);
```

Reply:
`selectedResourceUuid` is already formatted with the same syntax

Reply:
![Screenshot 2021-06-30 at 11 15 54](https://user-images.githubusercontent.com/12515407/123935507-94d05300-d994-11eb-8aea-c4cc1e5fd1ea.png)

Reply:
the "h" of uuid

It's on the `useResourceContext` repo that the "h" of the uuid format is created?